### PR TITLE
Grammar corrections in abc.rst

### PR DESCRIPTION
**Pull request by andresdelfino**

### CODE REVIEW

File: Doc/library/abc.rst [hunk 1]
```diff
@@ -18,10 +18,10 @@ see the PEP for why this was added to Python. (See also :pep:`3141` and the
 :mod:`numbers` module regarding a type hierarchy for numbers based on ABCs.)
 
 The :mod:`collections` module has some concrete classes that derive from
-ABCs; these can, of course, be further derived. In addition the
+ABCs; these can, of course, be further derived. In addition, the
 :mod:`collections.abc` submodule has some ABCs that can be used to test whether
-a class or instance provides a particular interface, for example, is it
-hashable or a mapping.
+a class or instance provides a particular interface, e.g.: if it is
```
Comment (Mariatta) [inline Doc/library/abc.rst]: While I agree this is an improvement, I'm wondering if "for example" is more easily understood to readers whose primary language is not English?

Comment (andresdelfino) [reply Doc/library/abc.rst]: e.g. like i.e. is pretty common in the documentation. But I can change it :) What I think it's really wrong is the "is it" vs. "if it is".

Comment (Mariatta) [reply Doc/library/abc.rst]: Yep "if it is" definitely is better :)

Comment (andresdelfino) [reply Doc/library/abc.rst]: Changed!

**Review by Mariatta — APPROVED** [review]

**Comment by Mariatta** [comment]
Thanks! miss-islington will take care the rest (I hope!)

**Pull request merged**